src/executables/commdb.c

```diff
@@ -1075,8 +1091,20 @@ process_batch_command (CSS_CONN_ENTRY * conn)
 
   if ((commdb_Arg_server_name) && (!commdb_Arg_halt_shutdown))
     {
-      pid = process_server_info_pid (conn, (char *) commdb_Arg_server_name, COMM_SERVER);
-      process_slave_kill (conn, (char *) commdb_Arg_server_name, commdb_Arg_shutdown_time, pid);
+      if (!commdb_Arg_single_node)
+	{
+	  pid = process_server_info_pid (conn, (char *) commdb_Arg_server_name, COMM_SERVER, SERVER_TYPE_ANY);
+	  process_slave_kill (conn, (char *) commdb_Arg_server_name, commdb_Arg_shutdown_time, pid);
```

Comment:
I know it's historic, but this function does not `kill`. It commands a graceful stop.

Reply:
True, and it is not only stopping the slave. However, there is an entire set of names that has the same problem and it would have to be renamed together. I'll make a low priority renaming task for this.

Reply:
Don't. As it would needlessly diverge even more from develop. :)